File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
 
+## Accessible PDF document
+
+This setting allows you to determine whether to preserve document structured tags in the converted PDF document for accessibility (508 or PDF/UA compliance) support. This property will set the title and description for images, diagrams and other objects in the generated PDF document. This information will be useful for people with vision or cognitive impairments who may not able to see or understand the object.
+
+The following code sample shows how to preserve document structured tags in the converted PDF document.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Open a PowerPoint Presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+    //Create an instance of ChartToImageConverter and assigns it to ChartToImageConverter property of Presentation.
+    pptxDoc.ChartToImageConverter = new ChartToImageConverter();
+    //Initialize the conversion settings.
+    PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+    // Enable a flag to preserve document structured tags in the converted PDF document.               
+    pdfConverterSettings.AutoTag = true;
+    //Convert the PowerPoint Presentation into PDF document.
+    using (PdfDocument pdfDocument = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings))
+    {
+        //Save the PDF document.
+        pdfDocument.Save("Sample.pdf");
+    }
+}
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+
+'Open a PowerPoint Presentation.
+Using pptxDoc As IPresentation = Presentation.Open("Sample.pptx")
+    'Create an instance of ChartToImageConverter and assigns it to ChartToImageConverter property of Presentation.
+    pptxDoc.ChartToImageConverter = New ChartToImageConverter()
+    'Initialize the conversion settings.
+    Dim pdfConverterSettings As PresentationToPdfConverterSettings = New PresentationToPdfConverterSettings()
+    'Enable a flag to preserve document structured tags in the converted PDF document.
+    pdfConverterSettings.AutoTag = True
+    'Convert the PowerPoint Presentation into PDF document.
+    Using pdfDocument As PdfDocument = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings)
+        'Save the PDF document.
+        pdfDocument.Save("Sample.pdf")
+    End Using
+End Using
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+
+//Load the PowerPoint presentation into stream.
+using (FileStream fileStreamInput = new FileStream("Sample.pptx", FileMode.Open, FileAccess.Read))
+{
+    //Open the existing PowerPoint presentation with loaded stream.
+    using (IPresentation pptxDoc = Presentation.Open(fileStreamInput))
+    {
+        //Instantiation of PresentationToPdfConverterSettings.
+        PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+        // Enable a flag to preserve document structured tags in the converted PDF document.               
```

Comment:
// Enable a flag to preserve structured document tags in the converted PDF document.